pyproject.toml

```diff
@@ -0,0 +1,69 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = ["setuptools", "wheel"]
+
+[project]
+authors = [{name = "SeedSigner", email = "author@example.com"}]
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent"
+]
+description = "Build an offline, airgapped Bitcoin signing device for less than $50!"
+name = "seedsigner"
+readme = "README.md"
+requires-python = ">=3.10"
+version = "0.7.0"
```

Comment:
I realize this was previously in setup.py, but I wonder if there's any way to mitigate the minor housekeeping required to keep this in sync with releases.

Can version just be omitted? We're not going to be releasing this as a pip package so specifying version here seems pointless (unless required by spec).

Reply:
Rebased the PR so that the version in the pyproject.toml file is now `0.8.0-rc1` like on the current dev branch.

Removing the version entirely is not possible as it results in the error `configuration error: `project` must contain ['version'] properties` when pip installing or creating a sdist via `python -b build .`.

Another option would to calculate the version dynamically based on repo tags via [setuptools-scm](https://pypi.org/project/setuptools-scm/)... maybe something for another PR, as it is a logical change and not just moving config settings into the pyproject.toml.